sailfishusermanagerinterface.h

```diff
@@ -12,10 +12,20 @@
 
 #include <QString>
 #include <QDBusArgument>
+#include <QDBusError>
 
 #define SAILFISH_USERMANAGER_DBUS_INTERFACE "org.sailfishos.usermanager"
 #define SAILFISH_USERMANAGER_DBUS_OBJECT_PATH "/"
 
+#define SailfishUserManagerErrorBusy "org.sailfishos.usermanager.error.busy"
+#define SailfishUserManagerErrorHomeCreateFailed "org.sailfishos.usermanager.error.homecreatefailed"
```

Comment:
I think errors should be TitleCased (or at least camelCased like methods are). I believe TitleCase is what's usually used on D-Bus. Also Error is usually written with capital E on D-Bus interfaces.
https://dbus.freedesktop.org/doc/dbus-specification.html#message-protocol-names-error

Reply:
Titlecased